cmd/build.go

```diff
@@ -194,7 +194,7 @@ func (b *buildCmd) createBuildTask() (*graph.Task, error) {
 	// TODO: create secrets
 	secrets := []*graph.Secret{}
 
-	return graph.NewTask(steps, secrets, b.opts.Registry, b.registryUser, b.registryPw, true)
+	return graph.NewTask(steps, secrets, b.opts.Registry, b.registryUser, b.registryPw, buildTimeoutInSec+pushTimeoutInSec, true)
```

Comment:
I guess the constant should be login + build + push + buffer, maybe we can just define it in its own constant.

Reply:
btw I made https://github.com/Azure/acr-builder/issues/340 as a separate item, don't have to tackle it in this PR but should make sure it doesn't regress